internal/playwright/config.go

```diff
@@ -75,8 +76,17 @@ func FromFile(cfgPath string) (Project, error) {
 	}
 
 	// Default project path
-	if p.Playwright.ProjectPath == "" {
-		return Project{}, fmt.Errorf("no project folder defined")
+	if p.Playwright.ProjectPath == "" && p.RootDir == "" {
+		return Project{}, fmt.Errorf("could not find 'rootDir' in config yml, 'rootDir' must be set to specify project files")
+	}
+	if p.Playwright.ProjectPath != "" {
```

Comment:
```suggestion
	if p.Playwright.ProjectPath != "" && p.RootDir == "" {
```

Otherwise you will get twice the deprecated message.